src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -121,14 +121,14 @@ public void testParsingErrors(@SysErr Capturable systemErr) throws Exception {
     @Test
     public void testWithMultipleChecksOne() throws Exception {
```

Comment:
Please go through the issue description again. There are a few ideal commits attached for reference.
You have to change test method names to match input filenames.